server/utils/commonLocale.json

```diff
@@ -22,6 +22,22 @@
         "agreePlanButton": "Agree plan"
       }
     },
+    "goalSummaryCard": {
+      "info": {
+        "aimToAchieve": "Aim to achieve in {{ monthsDifference }} months ({{ targetDate }})"
```

Comment:
Is this consistent with the new content?

Figma has different text, and two possible formats depending on what has been chosen.

Reply:
This is just the pre-emptive refactor work, I've updated it in the follow up PR